ConsentViewController/Classes/SPConsentManager.swift

```diff
@@ -328,13 +328,14 @@ import UIKit
 
     public func loadGDPRPrivacyManager(withId id: String, tab: SPPrivacyManagerTab = .Default, useGroupPmIfAvailable: Bool = false) {
         messagesToShow += 1
-        let usedId: String
-        if useGroupPmIfAvailable, let childPmId = storage.gdprChildPMId {
-            usedId = childPmId
-        } else {
-            usedId = id
+        var usedId: String = id
```

Comment:
Just a stylist consideration: nested ifs are not so readable, you can consider to flat them in only one.

Reply:
@carmelo-iriti totally makes sense